src/core/meshcop/dtls.cpp

```diff
@@ -438,7 +440,16 @@ ThreadError Dtls::MapError(int rval)
 
 void Dtls::HandleMbedtlsDebug(void *ctx, int level, const char *file, int line, const char *str)
 {
-    otLogInfoMeshCoP("%s:%04d: %s\r\n", file, line, str);
+#ifdef WINDOWS_LOGGING
+
+    if (strnlen(str, 512) != 512)
+    {
```

Comment:
To be fair, I'm not sure I've found them that useful either.  How about just removing the file and line print so we can avoid `WINDOWS_LOGGING` specific code here?